docs/data-structures/ceil_in_bst.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Ceil in Binary Search Tree
+slug: /data-structures/ceil_in_bst
+---
+
+## Introduction
+
+In this article, we will see how to find the ceil value in binary search tree(BST). We are given an Binary Search Tree(BST) and an element and we have to find the ceil value of that element in bst . In this type of tree(BST), left child is always smaller than its parent and right child is always greater than its parent.
+
+## Explanation 
+
+Ceil means smallest greater or equal value, if element is present then simply print that element otherwise print the smallest value which is greater than given value and if no greater value present then print null.
```

Comment:
```suggestion
Ceil means smallest greater or equal value if the element is present, then print that element; otherwise, print the smallest value greater than the given value. If no greater value is present, then print null.
```